models/engagements/supervision.py

```diff
@@ -29,6 +29,10 @@ class AbstractSupervision(BlurryPeriodMixin, BaseModel):
         blank=True
     )
 
+    description = models.TextField(
+        blank=True,
```

Comment:
Consider adding an attribute `null=True` as well because this will, although, successfully validate an empty string (`""`) but will also store it as `""` in database. Instead, store it as `NULL` in DB if an empty string is passed.
Reference: https://stackoverflow.com/a/13542137/11751642